report/UsabilityTest/UsabilityTest.tex

```diff
@@ -0,0 +1,49 @@
+\section(Usability testing)
+In order to determine if our application is easy to use and fulfils the requirements, we decided to perform usability testing of our application very early in the project.
+However due to the Covid-19 situation, we have been left with no options of performing usability testing in a safe environment and since we have not been able to perform the usability tests in a safe environment, we have been forced to not test the application.
```

Comment:
```suggestion
However, due to the Covid-19 situation, we have been left with no options of performing usability testing in a safe environment and since we have not been able to perform the usability tests in a safe environment, we have been forced to not test the application.
```